src/commands/ticker/index/processor.ts

```diff
@@ -336,6 +370,7 @@ export async function viewTickerChart(
     ...(days && { days: +days }),
     symbol,
     discordId,
+    isDominanceChart: JSON.parse(isDChart),
```

Comment:
`JSON.parse` sẽ throw exception nếu input pass vào là invalid string
```suggestion
    isDominanceChart: String(isDChart).toLowerCase() === "true",
```